src/dir.targets

```diff
@@ -94,8 +94,18 @@
   <Import Project="$(ToolsDir)publishtest.targets" Condition="Exists('$(ToolsDir)publishtest.targets')" />
 
   <PropertyGroup>
+    <IsInnerLoopTestProject Condition="'$(IsInnerLoopTestProject)'==''">True</IsInnerLoopTestProject>
```

Comment:
How would you default it? Would it be enforcing to have some category in every test project or rather default to InnerLoop and then:
<TestProjectCategory>$(TestProjectCategory);OuterLoop</TestProjectCategory>

but in this case how would you remove some category?

I feel like it adds additional complexity when we don't need it yet.